x-pack/plugins/enterprise_search/public/applications/app_search/components/engine/utils.ts

```diff
@@ -27,3 +27,11 @@ export const getEngineBreadcrumbs = (breadcrumbs: BreadcrumbTrail = []) => {
   const { engineName } = EngineLogic.values;
   return [ENGINES_TITLE, engineName, ...breadcrumbs];
 };
+
+/**
+ * Generate an API endpoint
+ */
+export const getEngineApiEndpoint = (endpoint: string): string => {
+  const { engineName } = EngineLogic.values;
+  return `/api/app_search/engines/${engineName}/${endpoint}`;
+};
```

Comment:
I see this was added as a part of 1d2e2de60b0d71402b438ef379ef7374eda3a016 but I'm not sure I understand why. Hey @constancecchen, old friend, possible this was accidentally included in your commit?

Reply:
Oh whoa! Thanks for catching this Byron, this was totally me, and I have no idea how. It should've been in a old local branch and not in my active staging, but seems to have got in somehow. I'll revert!

Reply:
https://github.com/elastic/kibana/pull/117242/commits/fd9f20de4a5a20ecafbf30d1146821ced94e7122